src/cpuinfo/SDL_cpuinfo.c

```diff
@@ -289,7 +289,7 @@ CPU_calcCPUIDFeatures(void)
                 /* Check to make sure we can call xgetbv */
                 if (c & 0x08000000) {
                     /* Call xgetbv to see if YMM (etc) register state is saved */
-#if defined(__GNUC__) && (defined(__i386__) || defined(__x86_64__))
+#if (defined(__GNUC__) || defined(__clang__)) && (defined(i386) || defined(__x86_64__))
```

Comment:
Please don't change `__i386__` back to `i386`

Reply:
Did this only to resolve the conflicts. I guess now the supplied patch is also deprecated.